cmd/kubetoken/kubetoken.go

```diff
@@ -17,13 +17,13 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"github.com/okteto/okteto/pkg/types"
 	"os"
 
+	"github.com/okteto/okteto/pkg/okteto"
```

Comment:
I'll try to remove as much dependency with the pk/okteto since we are getting the interfaces from there, but they should be declared by the file that will use them